### PR TITLE
Homework_4_vadim

### PR DESCRIPTION
**Pull request by VChernyshoff**

### CODE REVIEW

File: vchernyshoff/homework_4/olx_scraper/olx_scraper/items.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://docs.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+from scrapy import Field
```
Comment (PolyMaG) [inline vchernyshoff/homework_4/olx_scraper/olx_scraper/items.py:7]: You can use just one import

**Review by PolyMaG — APPROVED** [review]

File: vchernyshoff/homework_4/olx_scraper/olx_scraper/items.py (hunk 1 above)
Comment (VChernyshoff) [reply vchernyshoff/homework_4/olx_scraper/olx_scraper/items.py:7]: Thanks, I will

File: vchernyshoff/homework_4/olx_scraper/olx_scraper/spiders/goods_spider.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            good['seller_name'] = extract_with_css('h4 a::text')
+            good['seller_address'] = response.xpath(
+                '//*[@id="offeractions"]/div[4]/div[2]/div[1]/address/p/text()').get()
+            good['olx_delivery_availability'] = extract_with_css(
+                '.AdPageBox__link-button-caption::text')
+            self.count += 1
+
+            yield good
+        else:
+            raise CloseSpider('Exceeded maximum items')
```
Comment (DimaPoliuha) [inline vchernyshoff/homework_4/olx_scraper/olx_scraper/spiders/goods_spider.py]: No newline at the end of the file

**Review by DimaPoliuha — COMMENTED** [review]
Nice!:+1:

File: vchernyshoff/homework_4/olx_scraper/olx_scraper/output.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
...price": 56250, "currency": "грн.", "category": ["Объявление Киев", "Электроника Киев", "Аудиотехника Киев", "Прочая аудиотехника Киев"], "state": "Новый", "description": "EVOBOX Plus [Black] - новая караоке-система, новые возможности для Вашего домашнего досуга! А также, благодаря своей комплектации, может быть использована в караоке-клубах!", "date_of_creation": "в 18:36, 7 октября 2020", "views_count": 350, "id": "651377816", "photos_urls": ["https://ireland.apollo.olxcdn.com:443/v1/files/vegtxnjisy6f1-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/n06s8a3t0dq2-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/gza9r9a2zctj3-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/kjf26y0dzgqq3-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/qqnvhwwxg1ul2-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/w871was6ja5x2-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/z3h2pf77bar22-UA/image;s=1000x700", "https://ireland.apollo.olxcdn.com:443/v1/files/tmkhxac0fhq81-UA/image;s=1000x700"], "seller_name": "Вадим", "seller_address": null, "olx_delivery_availability": ""}
```
Comment (andrey-ned) [inline vchernyshoff/homework_4/olx_scraper/olx_scraper/output.json:1]: Our mentor wanna see price like float. It's not important.

**Review by andrey-ned — APPROVED** [review]

File: vchernyshoff/homework_4/olx_scraper/olx_scraper/spiders/goods_spider.py (hunk 2 above)
Comment (VChernyshoff) [reply vchernyshoff/homework_4/olx_scraper/olx_scraper/spiders/goods_spider.py]: fixed

**Pull request merged**